monkey/agent_plugins/payloads/ransomware/src/ransomware_options.py

```diff
@@ -23,7 +24,7 @@
 )
 
 _windows_absolute_path_regex = re.compile("^([A-Za-z]:(\\\\|/))")  # path starts like `C:\` OR `C:/`
-_windows_env_var_non_numeric_regex = re.compile(r"[A-Za-z#$'()*+,\-\.?@[\]_`\{\}~ ]")
+_windows_env_var_non_numeric_regex = re.compile(r"[A-Za-z#$'()*+,\-\.?@[\]_`\{\}~ ]]")
```

Comment:
I don't think this change has the desired effect. Did a test stop passing? Did you maybe mean

```suggestion
_windows_env_var_non_numeric_regex = re.compile(r"[A-Za-z#$'()*+,\-\.?@\[\]_`\{\}~ ]")
```